init-pants/action.yaml

```diff
@@ -66,12 +66,14 @@ runs:
       shell: bash
       run: |
         if ! command -v pants; then
-          if [[ ! -f ./get-pants.sh ]]; then
-            curl --proto '=https' --tlsv1.2 -fsSLO \
+          if [[ -f ./get-pants.sh ]]; then
+            ./get-pants.sh
+          else
+            curl --proto '=https' --tlsv1.2 -fsSLo /tmp/get-pants.sh \
```

Comment:
I guess /tmp/get-pants.sh is probably fine, but using mktemp or [`GITHUB_ACTION_PATH`](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) to get a truly unique path seems like better form. You never know what another action might do.